db/rdbms/src/main/resources/mapper/ProcessDefinitionMapper.xml

```diff
@@ -320,4 +320,147 @@
+    SELECT COUNT(*)
+    FROM (
+      SELECT pi.PROCESS_DEFINITION_ID
+      FROM ${prefix}PROCESS_INSTANCE pi
+      <where>
+        pi.STATE = 'ACTIVE'
+        <if test="authorizedResourceIds != null and !authorizedResourceIds.isEmpty()">
+          AND pi.PROCESS_DEFINITION_ID IN
+          <foreach collection="authorizedResourceIds" item="resourceId" open="(" separator="," close=")">
+            #{resourceId}
+          </foreach>
+        </if>
+        <if test="authorizedTenantIds != null and !authorizedTenantIds.isEmpty()">
+          AND pi.TENANT_ID IN
+          <foreach collection="authorizedTenantIds" item="tenantId" open="(" separator="," close=")">
+            #{tenantId}
+          </foreach>
+        </if>
+      </where>
+      GROUP BY pi.PROCESS_DEFINITION_ID, pi.TENANT_ID
+    )
+  </select>
+
+  <select id="processInstanceStatistics" parameterType="io.camunda.db.rdbms.read.domain.ProcessDefinitionInstanceStatisticsDbQuery"
+    resultType="io.camunda.search.entities.ProcessDefinitionInstanceStatisticsEntity">
+    SELECT pi.PROCESS_DEFINITION_ID,
+    pi.TENANT_ID,
+    MAX(pd.NAME) AS LATEST_PROCESS_DEFINITION_NAME,
+    CASE WHEN COUNT(DISTINCT pi.VERSION) > 1 THEN TRUE ELSE FALSE END AS HAS_MULTIPLE_VERSIONS,
+    SUM(CASE WHEN pi.NUM_INCIDENTS = 0 OR pi.NUM_INCIDENTS IS NULL THEN 1 ELSE 0 END) AS ACTIVE_INSTANCES_WITHOUT_INCIDENT_COUNT,
+    SUM(CASE WHEN pi.NUM_INCIDENTS > 0 THEN 1 ELSE 0 END) AS ACTIVE_INSTANCES_WITH_INCIDENT_COUNT
+    FROM ${prefix}PROCESS_INSTANCE pi
+    LEFT JOIN ${prefix}PROCESS_DEFINITION pd ON pd.PROCESS_DEFINITION_KEY = (
+      SELECT pi2.PROCESS_DEFINITION_KEY
+      FROM ${prefix}PROCESS_INSTANCE pi2
+      WHERE pi2.PROCESS_DEFINITION_ID = pi.PROCESS_DEFINITION_ID
+      ORDER BY pi2.VERSION DESC
+      LIMIT 1
+    )
+    <where>
+      pi.STATE = 'ACTIVE'
+      <if test="authorizedResourceIds != null and !authorizedResourceIds.isEmpty()">
+        AND pi.PROCESS_DEFINITION_ID IN
+        <foreach collection="authorizedResourceIds" item="resourceId" open="(" separator="," close=")">
+          #{resourceId}
+        </foreach>
+      </if>
+      <if test="authorizedTenantIds != null and !authorizedTenantIds.isEmpty()">
+        AND pi.TENANT_ID IN
+        <foreach collection="authorizedTenantIds" item="tenantId" open="(" separator="," close=")">
+          #{tenantId}
+        </foreach>
+      </if>
+    </where>
+    GROUP BY pi.PROCESS_DEFINITION_ID, pi.TENANT_ID
+    <include refid="io.camunda.db.rdbms.sql.Commons.keySetPageFilter"/>
+    <include refid="io.camunda.db.rdbms.sql.Commons.orderBy"/>
+    <include refid="io.camunda.db.rdbms.sql.Commons.paging"/>
+  </select>
+
+  <select id="processInstanceVersionStatisticsCount" parameterType="io.camunda.db.rdbms.read.domain.ProcessDefinitionInstanceVersionStatisticsDbQuery" resultType="long">
+    SELECT COUNT(*)
+    FROM
+    (SELECT
+    pi.PROCESS_DEFINITION_KEY
+    FROM
+    ${prefix}PROCESS_INSTANCE pi
+    LEFT JOIN
```

Comment:
💭 Using a LEFT JOIN will leave us with process instance entries that cannot be matched to process definitions by key, right? Does that make sense here? Would a simple JOIN be more appropriate? 🤔